libcxx/include/__ranges/to.h

```diff
@@ -81,6 +81,8 @@ template <class _Container, input_range _Range, class... _Args>
   static_assert(!is_const_v<_Container>, "The target container cannot be const-qualified, please remove the const");
   static_assert(
       !is_volatile_v<_Container>, "The target container cannot be volatile-qualified, please remove the volatile");
+  static_assert(
+      is_class_v<_Container>, "The target must be a class type");
```

Comment:
The changes should be clang-formatted. Ditto below.
```suggestion
  static_assert(is_class_v<_Container>, "The target must be a class type");
```

Reply:
FWIW, libstdc++ has switched to use `is_class_v || is_union_v` now, which exactly matches the currently standard wording.

Ping @Yuzhiy05: Is there any obstacle on writing tests or clang-formatting the files?